app/src/main/java/me/ccrama/redditslide/ColorPreferences.java

```diff
@@ -602,130 +606,207 @@ public int getColor(String s) {
     }
 
     public enum Theme {
```

Comment:
This block looks scary but all I did was swap the hardcoded themeType ints with the new enum values for consistency